src/Simplex_tree/include/gudhi/Simplex_tree/multi_filtrations/finitely_critical_filtrations.h

```diff
@@ -0,0 +1,161 @@
+#ifndef FINITELY_CRITICAL_FILTRATIONS_H_
+#define FINITELY_CRITICAL_FILTRATIONS_H_
+
+#include <iostream>
+#include <algorithm>
+#include <limits>
+
+namespace Gudhi::multiparameter::multi_filtrations{
+
+template<typename T=float>
+class Finitely_critical_multi_filtration : public std::vector<T> {
```

Comment:
Yes, furthermore they are not multi-critical for the moment, so `Multi_filtration_value` seems ok

Reply:
Oh no my bad, this can be confusing because multiparameter filtration values can be
 - finitely critical or
 - not finitely critical
This class is meant for the first kind, although it is only compatible with 1-critical filtrations for the moment (i.e. a simplex can only appear at one spot only).

Reply:
I've made an doc addition to the FinitelyCriticalFiltrations class to mention this